repos/system_upgrade/el7toel8/actors/kernel/checkinstalleddevelkernels/libraries/checkinstalleddevelkernels.py

```diff
@@ -27,8 +27,10 @@ def process():
         summary = ('DNF cannot produce a valid upgrade transaction when'
                    ' multiple kernel-devel packages are installed.')
         hint = ('Remove all but one kernel-devel packages before running Leapp again.')
-        commands = ([['yum', '-y', 'remove', '{n}-{v}.{r}'.format(
-            n=pkg.name, v=pkg.version, r=pkg.release)] for pkg in pkgs[:-1]])
+        all_but_latest_kernel_devel = pkgs[:-1]
+        packages = ['{n}-{v}.{r}'.format(n=pkg.name, v=pkg.version, r=pkg.release)
```

Comment:
```suggestion
        packages = ['{n}-{v}-{r}'.format(n=pkg.name, v=pkg.version, r=pkg.release)
```
There's a bug:
```
Remediation: [hint] Remove all but one kernel-devel packages before running Leapp again.
[command] yum -y remove kernel-devel-3.10.0.1062.4.3.el7
```
There should be a dash instead of period. This is the correct pkg name: `kernel-devel-3.10.0-1062.4.3.el7`.

Reply:
@AloisMahdal, I think it would be a great addition to the tests. If there's a remediation command, test that when applied, it fixes the reported issue.

Reply:
OAMG-3761 created based on my comment above.